app/components/my-profile.js

```diff
@@ -25,15 +24,6 @@ export default Component.extend({
   host: reads('iliosConfig.apiHost'),
   namespace: reads('iliosConfig.apiNameSpace'),
 
-  roles: computed('user.roles.[]', function() {
```

Comment:
?? removed?

Reply:
It wasn't being used. I could put it back.

Reply:
Ok, cool. No leave it out.